### DISCUSSION TITLE
addMesh doesnt show actual object

### DISCUSSION THREAD

**Discussion by maniast** [raisimTech/raisimLib#257] [Q&A]
Hello,
I am adding a obj file using addMesh and the mesh is imported as I can see the posx,y,z etc, however the actual object is not showing. Is there a way to fix this?

**Comment by jhwangbo** [comment]
can you share the mesh file?

**Reply by maniast** [reply]
https://www.turbosquid.com/3d-models/3d-stone-rock-1695868 is the obj file found here

**Comment by jhwangbo** [comment]
Unfortunately, we only support triangular mesh. The file you shared uses quadrilaterals.
